manifests/07-servicemonitor.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: insights-operator
+  namespace: openshift-insights
+spec:
+  endpoints:
+  - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+    interval: 30s
+    metricRelabelings:
+    - action: drop
+      regex: etcd_(debugging|disk|request|server).*
+      sourceLabels:
+      - __name__
+    port: https
+    scheme: https
+    tlsConfig:
+      # TODO where do you mount the service-ca.crt?
```

Comment:
How about this TODO? I see other operators having handled this https://github.com/openshift/cluster-authentication-operator/blob/3ccaf76d5a1f139800fd2360fd2d52af2edd4c35/manifests/0000_90_cluster-authentication-operator_02_servicemonitor.yaml#L20